Mathlib/Algebra/Group/WithOne/Defs.lean

```diff
@@ -6,7 +6,10 @@ Authors: Mario Carneiro, Johan Commelin
 import Mathlib.Algebra.Group.Defs
 import Mathlib.Data.Option.Basic
 import Mathlib.Logic.Nontrivial.Basic
-import Mathlib.Tactic.Common
+import Mathlib.Tactic.Coe
```

Comment:
I'm not sure if this is a good change; maybe the import was intentional. In some places, we deliberately import more tactics, to enable using them more easily. Can you look on zulip if there's a thread about this?